docs/FrontEndTests101.md

```diff
@@ -0,0 +1,112 @@
+### General information
+
+#### Where they live
+- Test directory contains all of the test code
+	- Configuration and helpers for test libraries live under `test/util/`, minus a few files that live directly under `test/`
+	- Unit tests and e2e tests live under each project directory
+	- Helpers for e2e tests are under `test/e2e/`
+	- Visual regression test config has it's own directory under `test/visual-regression`
+- Test commands are specified in `package.json`, as well as in vets-website's `README.md`
+- Configuration files for nightwatch and sass-lint live under `config/`
+- Scripts for running nightwatch tests live under `script/`
+
+#### When they run/ how they run
+There are 4 times tests can run:
+1. At any time during local development by running one of the npm scripts specified in `package.json`
+  - Any tests can be run during this time
+2. During the pre-commit hook
+  - Only linting is run automatically
+3. Before merging a branch to master via Jenkins
+  - Unit tests, e2e tests, and code coverage are automatically run during this time
+4. Before merging a branch to production via Jenkins
+  - Accessibility tests are automatically run during this time
+
```

Comment:
Right now the tests also run after merging to production. The devops team is planning on getting rid of the double test situation but not yet

Reply:
Ah good to know, thanks! Will update

Reply:
It adds some extra suspense during deployments lol. Sounds good